.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+
+node_js:
+  - lts/*
+
+cache: yarn
+
+before_deploy:
```

Comment:
Can we either add a noop `"test"` script in package.json or a noop `test` step here? travis currently erroring out

Reply:
Yeah, I know. I think I need to add script: true